### ISSUE TITLE
Second parameter to Ciphersweet constructor is not optional

### ISSUE THREAD

**Issue by thekevinbrown** [paragonie/ciphersweet-js#8]
```typescript
let engine = new CipherSweet(provider);
```
produces
```
(node:6304) UnhandledPromiseRejectionWarning: TypeError: Argument 2 must be an instance of backend
    at new CipherSweet (/Users/kevin/development/<project folder>/node_modules/ciphersweet-js/lib/ciphersweet.js:29:19)
... etc ...
```

I created #7 to attempt to make this match the docs.

**Comment by paragonie-security** [comment]
Thanks! This is fixed in `v1.2.2`.

**Comment by cj** [comment]
@paragonie-security I just updated to the latest version `2.0.1` and I am now getting this error.  The typescript says it is optional.

**Comment by paragonie-security** [comment]
That's a regression. Fixed in `v2.0.2`.
